controllers/scoring.py

```diff
@@ -183,13 +187,21 @@ def get_wants_piece_str(self) -> str:
 
     def get_cube_pickup(self) -> tuple[Pose2d, Rotation2d]:
         """Gets where to auto pickup cubes from"""
-        # can be changed for autonomous period?
+        if len(self.cube_queue) == 0:
```

Comment:
nit: use truthiness to check emptiness
```suggestion
        if not self.cube_queue:
```